health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/WorkflowCommentPopUp.js

```
@@ -0,0 +1,174 @@
+            }
+        );
+
+        setIsSubmitting(false);
+    };
+
+    return (
+        <>
+            <PopUp
+                onClose={onClose}
+                heading={t(heading)}
+                children={[
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)